index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Hello Microverse</title>
+</head>
+<body>
+  <div>Hello Microverse</div>
```

Comment:
- [ ] checking the project [requirements](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md), this heading was supposed to be an `<h1>` tag.  kindly change it to `<h1>` instead of `<div>`. for HTML semantic reasons.

Reply:
Of course! thank you. 👍